scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/JobOutput.java

```diff
@@ -81,48 +100,152 @@ public int compare(Task o1, Task o2) {
      * 
      * @param jobId id of the job representing this output
      */
-    public JobOutput(int jobId) {
+    public JobOutput(String jobId) {
         this.jobId = jobId;
         this.lines = new TreeMap<Task, List<String>>(TASK_FINISHED_TIME_COMPARATOR);
     }
+    
+    
+    public List<String> update(Task task, String out, OutputMode outputMode){
+        ArrayList<String> remoteVisuLines = new ArrayList<String>();
 
+        
+        if(this.outputMode != outputMode){
```

Comment:
Problem with formatting sometimes there is a space before brackets and curly braces and sometimes not.